service-tests/bitbucket.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+const {
+  isMetric,
+  isMetricOpenIssues
+} = require('./helpers/validators.js');
+
+const t = new ServiceTester({ id: 'bitbucket', title: 'BitBucket badges' });
+module.exports = t;
+
+
+// tests for issues endpoints
+
+t.create('issues-raw (valid)')
+  .get('/issues-raw/atlassian/python-bitbucket.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'issues',
+    value: isMetric
+  }));
+
+t.create('issues-raw (invalid)')
+  .get('/issues-raw/atlassian/not-a-repo.json')
+  .expectJSON({ name: 'issues', value: 'invalid' });
```

Comment:
How about displaying `repo not found` instead of `invalid`? Someone asked for repo which does not exist.

Reply:
1. This block might potentially catch errors other than repo doesn't exist

2. This behaviour is consistent with the output of other endpoints. e.g:

* https://img.shields.io/cpan/v/not-a-package.svg
* https://img.shields.io/pypi/v/blablabla/not-a-package.svg
* https://img.shields.io/packagist/v/blablabla/not-a-package.svg

Reply:
Okay, though is there another way to detect the "not found" case? Agree with @platan that it would be better to detect it and return a helpful message. It lets the user know where the problem lies.

Reply:
Ah, didn't see this: https://github.com/badges/shields/pull/1315#discussion_r154759969